andrey_n/homework_4/olx_scrap/olx/spiders/example.py

```diff
@@ -0,0 +1,86 @@
+import scrapy
+from scrapy.exceptions import CloseSpider
+
+from olx.items import OlxItem
+
+
+class OlxSpider(scrapy.Spider):
+    '''
+    olx spider for scraping data
+    '''
```

Comment:
All docstrings should be in triple double-quotes: """Docstring"""
https://www.python.org/dev/peps/pep-0257/

Reply:
fixed